text/0059-token-metadata-document.md

```diff
@@ -0,0 +1,504 @@
+  "RAD",
+  "RARE",
+  "RARI",
+  "RAY",
+  "RBC",
+  "REN",
+  "REPV2",
+  "REQ",
+  "RLC",
+  "RNDR",
+  "ROOK",
+  "RPL",
+  "RUNE",
+  "SAMO",
+  "SAND",
+  "SBR",
+  "SC",
+  "SCRT",
+  "SCRT.S",
+  "SDN",
+  "SGB",
+  "SHIB",
+  "SNX",
+  "SOL",
+  "SOL.S",
+  "SPELL",
+  "SRM",
+  "STEP",
+  "STG",
+  "STORJ",
+  "STX",
+  "SUPER",
+  "SUSHI",
+  "SXP",
+  "SYN",
+  "T",
+  "TBTC",
+  "TEER",
+  "TLM",
+  "TOKE",
+  "TRIBE",
+  "TRU",
+  "TRX",
+  "TRX.S",
+  "TVK",
+  "UMA",
+  "UNFI",
+  "UNI",
+  "USD.HOLD",
+  "USD.M",
+  "USDC",
+  "USDT",
+  "UST",
+  "WAVES",
+  "WBTC",
+  "WETH",
+  "WOO",
+  "XBT.M",
+  "XCN",
+  "XETC",
+  "XETH",
+  "XLTC",
+  "XMLN",
+  "XREP",
+  "XRT",
+  "XTZ",
+  "XTZ.S",
+  "XXBT",
+  "XXDG",
+  "XXLM",
+  "XXMR",
+  "XXRP",
+  "XZEC",
+  "YFI",
+  "YGG",
+  "ZAUD",
+  "ZCAD",
+  "ZEUR",
+  "ZGBP",
+  "ZJPY",
+  "ZRX",
+  "ZUSD"
+]
+```
+
+Coinbase lists over 9000 tokens with letters, numbers, mixed case, and as many as 8 characters: `BabyDoge, LYXe, vBUSD, 7S, DRAGACE, TRINITY`.
+Coinbase also lists the tickers `OXD V2`, `$ZOOM`, which would not be allowed in the above specification due to the space character and `$` character.
+
+We do not recommend that space or `$` become allowed characters, as this creates the opportunity for confusion.
+
+# Prior art
+[prior-art]: #prior-art
+
+We considered the metadata used in both Cardano native assets and Algorand standard assets when considering this proposal.
+We also considered how tokens are assigned metadata in the Ethereum ecosystem.
+
+In the Ethereum ecosystem, most tokens (fungible and non-fungible) have logos, but they are typically stored on centralized servers, and referred to via URLs.
+
+In [EIS-2569](https://eips.ethereum.org/EIPS/eip-2569) it was proposed to actually store SVG data on chain.
+
+Typically, wallets like MetaMask use services like Infura to access names and logos of tokens. There is not a standard way outside of these
+services to get this data for an arbitrary ERC-20.
+
+In Algorand, any standard asset that is created has [required parameters](https://developer.algorand.org/docs/get-details/asa/#asset-parameters)
+
+> These eight parameters can *only* be specified when an asset is created.
+>
+>    Creator (required)
+>    AssetName (optional, but recommended)
+>    UnitName (optional, but recommended)
+>    Total (required)
+>    Decimals (required)
+>    DefaultFrozen (required)
+>    URL (optional)
+>    MetaDataHash (optional)
+
+In Cardano, native assets can be created before being [registered with the token registry](https://developers.cardano.org/docs/native-tokens/token-registry/How-to-prepare-an-entry-for-the-registry-NA-policy-script). Only a name is required to create the asset. When subsequently registering an asset, a "description" is required. Then a ticker, url, logo, and decimals are all considered optional. In the Cardano registry, metadata can be deleted and updated, but it requires a signature from the creator.
```

Comment:
:upside_down_face:  nit
The quotes around description felt out of place when ticker, url, logo, etc were naked
```suggestion
In Cardano, native assets can be created before being [registered with the token registry](https://developers.cardano.org/docs/native-tokens/token-registry/How-to-prepare-an-entry-for-the-registry-NA-policy-script). Only a name is required to create the asset. When subsequently registering an asset, a description is required. Then a ticker, url, logo, and decimals are all considered optional. In the Cardano registry, metadata can be deleted and updated, but it requires a signature from the creator.
```